tiling.js

```diff
@@ -3797,20 +3818,30 @@ function cycleWindowWidth(metaWindow) {
     metaWindow.move_resize_frame(true, targetX, frame.y, targetWidth, frame.height);
 }
 
-function cycleWindowHeight(metaWindow) {
+function cycleWindowHeight(metawindow) {
+    return cycleWindowHeightDirection(metawindow, CycleWindowSizesDirection.FORWARD);
+}
+
+function cycleWindowHeightBackwards(metawindow) {
+    return cycleWindowHeightDirection(metawindow, CycleWindowSizesDirection.BACKWARDS);
+}
+
+function cycleWindowHeightDirection(metaWindow, direction) {
     let steps = Settings.prefs.cycle_height_steps;
     let frame = metaWindow.get_frame_rect();
 
     let space = spaces.spaceOfWindow(metaWindow);
     let i = space.indexOf(metaWindow);
 
+    let findFn = direction === CycleWindowSizesDirection.FORWARD ? Lib.findNext : Lib.FindPrev;
```

Comment:
typo, should be:

```
    let findFn = direction === CycleWindowSizesDirection.FORWARD ? Lib.findNext : Lib.findPrev;
```

Notice the non-capitalised `Lib.findPrev`

Reply:
ouch!

I've realized I had only updated the height cycling function -- not only was it wrong, but incomplete ...

Good thing you tested it again!

Reply:
27e5c927318a8dfd4c7a0534d6051b7bb03ef1e6 should fix it

(I'll test as soon as I can restart my session)

Reply:
Done

Reply:
Cool, will give it a another test later on today and then merge into develop (and do I release with both your PRs).

Cheers.